tests/kani/Intrinsics/bswap.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+// Check that `bswap` returns the expected results
```

Comment:
Can you please state here what that result is? `bswap` is not the most intuitive name. It could be bit / byte swap. I had to google to check exactly what this does.